products/gkehub/api.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: GKEHub
+display_name: GKEHub
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://gkehub.googleapis.com/v1beta1/
```

Comment:
Why not use the v1 API?

Reply:
Since it's not publicly documented I wasn't sure if it was really GA or not.  If you're sure it is I can get it into both providers, no problem.

Reply:
The v1 API is GA.